lib/conjur/api/resources.rb

```diff
@@ -166,5 +166,19 @@ def resources opts = {}
     def global_privilege_permitted? privilege
       resource(GLOBAL_PRIVILEGE_RESOURCE).permitted? privilege
     end
+
+    def resources_permitted? kind, identifiers, privilege
+      options = {
+        privilege: privilege,
+        identifiers: identifiers
```

Comment:
Sorry, as I said on the `authz` PR, I made that change locally and forgot to push it. All my local testing so far has been using source directories mounted into my sandbox.